AUTHORS

```diff
@@ -16,26 +16,18 @@ Contributors
 CUSE has been written by Tejun Heo <teheo@suse.de>. Furthermore, the
 following people have contributed patches (autogenerated list):
 
+Alexander <aleksandr.rvachev@eltex-co.ru>
 Anatol Pomozov <anatol.pomozov@gmail.com>
-Antonio SJ Musumeci <trapexit@spawn.link>
-Christopher Harrison <ch12@sanger.ac.uk>
+Bill Zissimopoulos <billziss@navimatics.com>
+Carl Edquist <edquist@cs.wisc.edu>
 Csaba Henk <csaba.henk@creo.hu>
```

Comment:
This commit looks really weird. I didn't look at the others yet and won't have time this evening, but  from this commit it looks like something went wrong.